features/car/classes.feature

```diff
@@ -17,13 +17,13 @@ Feature: Car - Mode flag
             | cd    | primary |       |
 
         When I route I should get
-            | from | to | route       | turns                                              | classes      |
-            | a    | d  | ab,bc,cd,cd | depart,notification right,notification left,arrive | ,ferry,,     |
-            | d    | a  | cd,bc,ab,ab | depart,notification right,notification left,arrive | ,ferry,,     |
-            | c    | a  | bc,ab,ab    | depart,notification left,arrive                    | ferry,,      |
-            | d    | b  | cd,bc,bc    | depart,notification right,arrive                   | ,ferry,ferry |
-            | a    | c  | ab,bc,bc    | depart,notification right,arrive                   | ,ferry,ferry |
-            | b    | d  | bc,cd,cd    | depart,notification left,arrive                    | ferry,,      |
+            | from | to | route       | turns                                              | classes                  |
+            | a    | d  | ab,bc,cd,cd | depart,notification right,notification left,arrive | [[]],[[ferry]],[[]],[[]] |
```

Comment:
I like this `[]` approach. We should think about doing this on all outputs for intersections.